pinc/ProjectTransition.inc

```diff
@@ -195,7 +190,7 @@ class ProjectTransition
      * It should perhaps also be the only place that modifies these columns:
      *     modifieddate checkedoutby postproofer postcomments
      */
-    public function do_state_change($project, $who, $extras)
+    public function do_state_change(Project $project, string $who, $extras): string
```

Comment:
Can we say that $extras is an array?

Reply:
Indeed as all callers pass an array. Updated in the latest patch.